src/rosdiscover/interpreter/model.py

```diff
@@ -28,9 +28,9 @@ def register(package: str,
 
     @staticmethod
     def find(package: str, name: str) -> 'Model':
-        try:
+        if (package, name) in Model._models.keys():
```

Comment:
there shouldn't be a need for `.keys()` here. `(package, name) in Model._models` should suffice.

Reply:
Done.